angular/src/lib/drawer/drawer.component.ts

```diff
@@ -0,0 +1,60 @@
+ /** @component drawer */
+import { Component, Input, Output, EventEmitter, HostBinding } from '@angular/core';
+
+export type DrawerOpenFromDirection = 'right' | 'top' | 'bottom';
+export type DrawerSize = 'medium' | 'large';
+
+@Component({
+  selector: 'md-drawer',
+  template: `
+    <ng-content></ng-content>
+    <md-icon className="md-drawer__close" name="cancel_14" ariaLabel="{{closeAriaLabel}}" (click)="close()" ></md-icon>
+  `,
+  styles: []
+})
+export class DrawerComponent {
+
+  private _visible: Boolean = false;
+  public color: string = 'blue';
+
+
+  /** @option css class names | '' | '' */
```

Comment:
Are we using @prop or @option in angular? Also duplicated defaults and options below don't have defaults with pipe.  Comment tab from left to right appears inconsistent.

Reply:
@prop

Reply:
I guess there are 177 instances of @option we need to fix then

Reply:
corrected this one